dbfit-java/core/src/main/java/dbfit/api/DBEnvironment.java

```diff
@@ -189,4 +189,6 @@ void connectUsingFile(String filePath) throws SQLException, IOException,
             FileNotFoundException;
 
     DbStoredProcedureCall newStoredProcedureCall(String name, DbParameterAccessor[] accessors);
+
+    String buildFunctionCall(String procOrFuncName, int numberOfParameters);
```

Comment:
What about starting to separate it now with a different interface for the vendor API under `api.vendor` that `AbstractDbEnvironment` also implements?